plugins/woocommerce/tests/e2e-pw/tests/customize-store/assembler/font-picker.spec.js

```diff
@@ -40,7 +40,22 @@ const slugFontMap = {
 		'System Sans-serif',
 };
 
+const skipTestIfWPIs64 = () => {
+	const skipMessage = `Skipping this test because on WordPress 6.4, because the font picker is not available`;
+
+	test.skip( () => {
+		const shouldSkip = process.env.WP_ENV_CORE.includes( '6.4' );
```

Comment:
As I expected, this line would fail in WP latest (v6.5) because we don't use `WP_ENV_CORE` for WP Latest. I would suggest two ways to fix this:

First, add a condition to check if `process.env.WP_ENV_CORE` exists. So you'll have these two as the skip condition:

```js
process.env.WP_ENV_CORE && process.env.WP_ENV_CORE.includes('6.4')
```

Second, I would suggest to not have a `skipTestIfWPIs64` function at all. Instead, we can simplify the skip into just one line like this:

```js
test.describe( 'Assembler -> Font Picker', () => {

    test.skip( process.env.WP_ENV_CORE && process.env.WP_ENV_CORE.includes('6.4'), 'Skipping, font picker not available in WP 6.4');

    // ...
```

For a little context: you may see some E2E tests where there was a dedicated skip function (much like how you did with the `skipTestIfWPIs64()` function). I did those, and I kinda regretted it :sweat_smile: I did it so that the skip message would be printed on the allure reports. Because Allure reports for some reason don't output the skip message using the one-line approach that I suggested above. Since we're aiming to only temporarily disable this test on WP 6.4, then there's no need to elaborate the skip mechanism that way.  Hope this helps :)

Reply:
Thanks for the review! I changed the code following your instruction [fb9ef60](https://github.com/woocommerce/woocommerce/pull/47182/commits/fb9ef60d5f3a13e7fdd51b5c214b56a5c323a9c4)!

I'm going to mark this PR ready to be reviewed!

Reply:
Your recent change looks correct! Thanks :) Marking this as resolved now. I could test this on my local against WP L-1 to see if it would be correctly skipped.